shuup/front/apps/registration/notify_events.py

```diff
@@ -13,14 +14,15 @@
 from shuup.notify.base import Event, Variable
 from shuup.notify.script_template.factory import \
     generic_send_email_script_template_factory
-from shuup.notify.typology import Email, Model
+from shuup.notify.typology import Email, Model, URL
 
 
 class RegistrationReceived(Event):
     identifier = "registration_received"
     name = _("Registration Received")
     customer = Variable(_("Customer"), type=Model("shuup.Contact"))
     customer_email = Variable(_("Customer Email"), type=Email)
+    activation_url = Variable(_("Activation URL"), type=URL, required=False)
```

Comment:
Missing translations. 1.2 should be fully translated to at least English, Finnish and Swedish, so please update the corresponding PO files too.